build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -99,6 +99,8 @@ spec:
             {{- end }}
             # SKIP_CNI_BINARIES takes in values as a comma separated list of
```

Comment:
notice how you separated this comment with from the corresponding variable definition below